client/src/components/App/Content/TabManagement/TabManagement.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import rtl from 'jss-rtl';
+import { create } from 'jss';
+import { jssPreset } from '@material-ui/styles';
+import { Paper, Tabs, Tab, Card, StylesProvider } from '@material-ui/core';
+
+import TabObj from 'models/TabObj';
+
+import useStyles from './TabManagementStyles';
+
+const defaultTab: TabObj = {
+    id: 0,
+    name: 'פרטים אישיים'
+};
+
+const jss = create({ plugins: [...jssPreset().plugins, rtl()] })
+
+const TabManagement: React.FC = (): JSX.Element => {
+    const classes = useStyles({});
+
+    const [currentTab, setCurrentTab] = React.useState<TabObj>(defaultTab);
+
+  const handleTabChange = (event: React.ChangeEvent<{}>, selectedTab: number) => {
+    setCurrentTab({
+        id: selectedTab,
+        name: tabs[selectedTab].name
+    });
+  };
+
+    const tabs: TabObj[] = [
+        defaultTab,
```

Comment:
consider putting the defaultTab as is in here (and not in const),
and setting the default tab to be tabs[0]
( that will prevent some copy-pasting if the default tab changes)

Reply:
the default tab can can also be a const that points to tabs[0] or something like that